app/controllers/users_controller.rb

```diff
@@ -1,8 +1,26 @@
 class UsersController < ApplicationController
+  before_action :set_user, only: %i(show edit update destroy)
+
+  def index
+    @users = User.order(created_at: :desc)
+  end
+
+  def show
+    return if @user
+
+    redirect_to users_path, alert: t("users.not_found")
+  end
```

Comment:
```suggestion
  def show; end
```
Đưa logic này vào trong hàm load_user luôn e nhé, để k cần xử lý thêm trong các action